src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -87,6 +87,8 @@
     "d:transactions/alert@none": PREFIX + 133,
     "s:transactions/alert@none": PREFIX + 134,
     "g:transactions/alert@none": PREFIX + 135,
+    "t:transactions/duration@millisecond": PREFIX + 136,
+    "t:transactions/measurements.lcp@millisecond": PREFIX + 137,
```

Comment:
what's the significance of the prefix here? I've made it `t` just to make it unique but I'm not sure if it should be something else